skfda/preprocessing/dim_reduction/feature_extraction/_fda_feature_union.py

```diff
@@ -0,0 +1,127 @@
+    name separated by a '__'. A transformer may be replaced entirely by
+    setting the parameter with its name to another transformer,
+    or removed by setting to 'drop'.
+
+    Parameters:
+        transformer_list: list of tuple
+            List of tuple containing `(str, transformer)`. The first element
+            of the tuple is name affected to the transformer while the
+            second element is a scikit-learn transformer instance.
+            The transformer instance can also be `"drop"` for it to be
+            ignored.
+        n_jobs: int
+            Number of jobs to run in parallel.
+            ``None`` means 1 unless in a :obj:`joblib.parallel_backend`
+            context.
+            ``-1`` means using all processors.
+            The default value is None
+        transformer_weights: dict
+            Multiplicative weights for features per transformer.
+            Keys are transformer names, values the weights.
+            Raises ValueError if key not present in ``transformer_list``.
+        verbose: bool
+            If True, the time elapsed while fitting each transformer will be
+            printed as it is completed. By default the value is False
+        array_output: bool
+            indicates if the transformed data is requested to be a NumPy array
+            output. By default the value is False.
+
+    Examples:
+    Firstly we will import the Berkeley Growth Study data set
+    >>> from skfda.datasets import fetch_growth
+    >>> X = fetch_growth(return_X_y=True)[0]
+
+    Then we need to import the transformers we want to use
+    >>> from skfda.preprocessing.dim_reduction.feature_extraction import (
+    ...     FPCA,
+    ...     FdaFeatureUnion,
+    ... )
+    >>> from skfda.representation import EvaluationTransformer
+
+    Finally we apply fit and transform
+    >>> union = FdaFeatureUnion(
+    ...     [
+    ...        ("eval", EvaluationTransformer()),
+    ...        ("fpca", FPCA()),
+    ...     ],
+    ...     array_output=True,
+    ... )
+    >>> transformed_data = union.fit_transform(X)
+    >>> transformed_data
+        [[ 81.3       ,  84.2       ,  86.4       , ..., 105.84283261,
+            -34.60733887, -14.97276458],
+           [ 76.2       ,  80.4       ,  83.2       , ..., -11.42260839,
+            -17.01293819,  24.77047871],
+           [ 76.8       ,  79.8       ,  82.6       , ..., -33.81180503,
+            -23.312921  ,   7.67421522],
+           ...,
+           [ 68.6       ,  73.6       ,  78.6       , ..., -19.49404628,
+             12.76825883,   0.70188222],
+           [ 79.9       ,  82.6       ,  84.8       , ...,  19.28399897,
+             31.49601648,   6.54012077],
+           [ 76.1       ,  78.4       ,  82.3       , ...,  17.71973789,
+             27.7332045 ,  -1.70532625]]
+
+    We can also concatenate the result with the
+    original data on a Pandas DataFrame.
+    >>> from pandas.core.frame import DataFrame
```

Comment:
Don't concatenate it manually! Show an example using `FdaFeatureUnion`.

Reply:
I just noticed that EvaluationTransformer() actually gives the original data, and so we do not need to concatenate it again. I will add to the example another transformer like SRSF()